tasks/pull-disk-images/README.md

```diff
@@ -0,0 +1,13 @@
+# pull-disk-images
+
+Tekton task to pull disk images from quay using oras pull.
+
+The disk images are stored in a workspace, and they are expected to be quite large. Due to this,
+the data workspace that is attached to this task should be quite large, somewhere around 100G.
```

Comment:
Some images are 140GB actually. Maybe no need to go into specifics.

Reply:
I think it is worth mentioning specifics because a workspace so large is unusual. I meant to just pick a number that was 10^x. I can try rewording it

Reply:
modified